osmnx/buildings.py

```diff
@@ -165,7 +165,7 @@ def create_buildings_gdf(polygon=None, north=None, south=None, east=None,
                 nodes = result['nodes']
                 try:
                     polygon = Polygon([(vertices[node]['lon'], vertices[node]['lat']) for node in nodes])
-                except:
+                except Exception as e:
```

Comment:
Yeah. I'm thinking about adding explicit errors, but it'll be a chunk of work. For now I'm just considering going through to at least define a general Exception e. If you are looking for more specific error catching downstream, PRs always welcome!